addons/nuxeo-platform-pdf-utils/src/main/java/org/nuxeo/ecm/platform/pdf/PDFPageNumbering.java

```diff
@@ -105,12 +106,12 @@ public Blob addPageNumbers(int inStartAtPage, int inStartAtNumber, String inFont
             if (StringUtils.isBlank(inFontName)) {
                 font = PDType1Font.HELVETICA;
             } else {
-                font = PDType1Font.getStandardFont(inFontName);
+                font = PDFUtils.getStandardType1Font(inFontName);
                 if (font == null) {
-                    font = new PDType1Font(inFontName);
+                    throw new NuxeoException("Not a standard font: " + inFontName);
                 }
             }
-            allPages = doc.getDocumentCatalog().getAllPages();
+            allPages = Lists.newArrayList(doc.getDocumentCatalog().getPages());
```

Comment:
`getPages()` returns an iterator, and there's no one-method way of doing the conversion to a list in pure Java. But I could use Apache Commons Collections instead of Guava here.
Note that in this patch I wanted to do the minimum changes possible to separate migration tasks from refactorings/cleanups.

Reply:
Understood, 👍

Reply:
Updated